Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'launchy', '~> 2.4.3'
   gem 'rails-controller-testing'
   gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
+  gem 'friendly_id', '~> 5.4.0'
```

Comment:
Would you know how to do this with a Gem? Do you how this gem works and what trade-offs you are making?

Reply:
I think the trade-offs are not complete control. The positives are they make it pretty easy. I think with google I could do it without the gem. I started to then run into issues with duplicate slugs then found this solution.